packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.style.scss

```diff
@@ -207,7 +207,7 @@
         }
     }
 
-    &:hover {
+    &:hover:not(&_isNotAvailable) {
```

Comment:
Please try to avoid `:not` if possible.
First, you define common styles, then create a mods class name and redefine styles according to the current case.

Reply:
Ready!